src/docfx/restore/Restore.cs

```diff
@@ -2,19 +2,86 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Diagnostics;
+using System.IO;
 using System.Threading.Tasks;
 
 namespace Microsoft.Docs.Build
 {
     internal static class Restore
     {
+        private static readonly string s_restoreDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".docfx", ".git");
```

Comment:
`.git` -> `git`, `.git` suggests it is a git repo, but it is just a container of git repos.